src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -421,6 +430,7 @@ class ChatSwitcherView extends React.Component {
             .map(personalDetail => ({
                 text: personalDetail.displayName,
                 alternateText: personalDetail.login,
+                participants: [personalDetail.login],
```

Comment:
I'm unsure of why we need `participants` at all if every report has one avatar per participant we could just check the number of `icons` we have.

Reply:
Done